vhdbuilder/packer/components.json

```diff
@@ -393,8 +388,7 @@
     {
       "downloadURL": "mcr.microsoft.com/azure-policy/policy-kubernetes-webhook:*",
       "amd64OnlyVersions": [
-        "prod_20210714.1",
```

Comment:
prod_20210909.1 was not used in RP for a while 

{{- else if eq .component "azure-policy-webhook" -}}
  {{- if semverCompare ">=1.18.0" .version -}}prod_20211230.1
  {{- else -}}prod_20210714.1
  {{- end -}}